src/services/search.js

```diff
@@ -39,10 +39,11 @@ export const searchUsers = async (keywords, offset, limit) => {
   return res.data;
 };
 
-export const getDiscoverItems = async (offset, limit) => {
+export const getDiscoverItems = async (offset, limit, category, viewTradable) => {
   const params = {
     offset,
     limit,
+    category,
```

Comment:
* Does category=null cause endpoint to return all items, or does it throw a bad request err?
* What about the `viewTradable` argument? Seems like its not being passed into the request.